core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -189,16 +189,22 @@ contract Liquidatable is PricelessPositionManager {
         address sponsor,
         FixedPoint.Unsigned calldata collateralPerToken,
         FixedPoint.Unsigned calldata maxTokensToLiquidate
-    ) external fees() onlyPreExpiration() returns (uint liquidationId) {
+    )
+        external
+        fees()
+        onlyPreExpiration()
+        returns (
+            uint liquidationId,
+            FixedPoint.Unsigned memory tokensLiquidated,
+            FixedPoint.Unsigned memory finalFeeBond
+        )
+    {
         // Retrieve Position data for sponsor
         PositionData storage positionToLiquidate = _getPositionData(sponsor);
 
-        FixedPoint.Unsigned memory tokensToLiquidate = FixedPoint.min(
-            maxTokensToLiquidate,
-            positionToLiquidate.tokensOutstanding
-        );
+        tokensLiquidated = FixedPoint.min(maxTokensToLiquidate, positionToLiquidate.tokensOutstanding);
         // TODO: Limit liquidations from being too small or very close to 100% without being exactly 100%.
-        FixedPoint.Unsigned memory ratio = tokensToLiquidate.div(positionToLiquidate.tokensOutstanding);
+        FixedPoint.Unsigned memory ratio = tokensLiquidated.div(positionToLiquidate.tokensOutstanding);
```

Comment:
Do you think its also worth returning `liquidatedCollateral`? Since this could be different from what the user is expecting due to rounding errors?

Reply:
Hmmm, maybe, but why would the caller care exactly how much collateral they were liquidating as long as it's below their limit? I only push back a little because almost all the values we currently return (here and in other methods) are directly related to balance transfers to and from the user's wallet, so liquidatedCollateral would be a little different.

Reply:
> I only push back a little because almost all the values we currently return (here and in other methods) are directly related to balance transfers to and from the user's wallet, so liquidatedCollateral would be a little different.

Fair point, I see no reason to add